.circleci/config.yml

```diff
@@ -34,6 +34,8 @@ commands:
       - utils/checkout-with-mise
       - run:
           name: "simulate << parameters.task >>"
+          environment:
+            ETH_RPC_URL: << pipeline.parameters.l1_mainnet_rpc_url >>
```

Comment:
I think this is not right, because we might be simulating on sepolia?

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/629/commits/e00e4dfa97eb42201cfdad64472349677246a048